api/models/recruitDao.js

```diff
@@ -0,0 +1,142 @@
+const { dataSource } = require('./data-source')
+
+const checkCategory = async(categoryId) => {
+    const [result] = await dataSource.query(
+        `SELECT EXISTS(
+            SELECT *
+            FROM categories
+            WHERE id IN (?)
+        )as a
```

Comment:
```suggestion
        ) as does_exist
```
a 보다는 명확한 네이밍이 좋을 것 같아요.